### DISCUSSION TITLE
Creating a new profile

### DISCUSSION THREAD

**Discussion by carolscliar** [zowe/zowe-cli#2224] [Q&A] [answered]
Hello, i am trying to start to IBM course for COBOL, but i'm finding really hard to create a ZOWE profile, i already have the ID and PASSWORD provided by IBM, i saw a lot of information but nothing seems to work. I have the 7.28.2 version

**Comment by adam-wolfe** [comment, marked as answer]
If you have something like this in ~/.zowe/zowe.config.json (sample from the OMP COBOL programming course):

```json
{
    "$schema": "./zowe.schema.json",
    "profiles": {
        "LearnCOBOL": {
            "properties": {
                "host": "204.90.115.200"
            },
            "profiles": {
                "zosmf": {
                    "type": "zosmf",
                    "properties": {
                        "port": 10443
                    }
                }
            },
            "secure": [
                "user",
                "password"
            ]
        },
        "base": {
            "type": "base",
            "properties": {
                "rejectUnauthorized": false
            },
            "secure": []
        }
    },
    "defaults": {
        "zosmf": "LearnCOBOL.zosmf",
        "base": "base"
    },
    "plugins": []
}
```

You would likely need to change the values for `host` and `port`. 

In Zowe CLI, you would then run the `zowe config secure --gc` command to enter a user ID and password.

**Comment by carolscliar** [comment]
When a put this in the terminal: `zowe config secure --gc` i have this message: No secure properties found in your config

**Reply by adam-wolfe** [reply]
Have you been given a `zowe.config.json` file? If you have, the file should be copied into the ~/.zowe directory. 

Otherwise, you could create a v1 profile using the `zowe profiles create zosmf-profile`.

**Comment by carolscliar** [comment]
Hi, thanks for the answered ! I manage to go to this part, but now i have this error when i try to open the file, do you know what can i do ? Thank you
![Screenshot 2024-08-09 155542](https://github.com/user-attachments/assets/a2618f0b-75d1-4bdc-aebc-c533717a1a97)

**Reply by carolscliar** [reply]
And it is supposed to look like this: 
![Screenshot 2024-08-09 155909](https://github.com/user-attachments/assets/92aa3218-8386-4cc0-91bf-e4aaf656df3a)

**Reply by adam-wolfe** [reply]
It looks like you'll need to submit the PDSBUILD or JCLSETUP jobs to get all of the data sets. Do the instructions tell you where to find the PDSBUILD and JCLSETUP members?

**Reply by carolscliar** [reply]
Yes, but i need to first find the "HELLO" file, but it doesn't show, it is suppose to be like this: 
![Screenshot 2024-08-09 160940](https://github.com/user-attachments/assets/173bc8a6-c17d-4322-ba10-68aaa5bf8b45)

**Comment by zFernand0** [comment]
For further assistance with the Cobol Programming course, you may have a better turnaround by reaching out in the Cobol-Programming-Course channel in the OMP Slack space.
https://openmainframeproject.slack.com/archives/C011NE32Z1T
